pallets/gear/src/tests.rs

```diff
@@ -1105,7 +1101,7 @@ fn auto_reply_out_of_rent_mailbox() {
         assert!(utils::is_active(program_id));
 
         let user1_balance = Balances::free_balance(USER_1);
-        assert_balance(program_id, value, 0u128);
+        assert_balance(program_id, value + ed, 0u128);
```

Comment:
ed shouldn't be free balance, it should be locked (as well for proper informationing on observers like polkajs)

Reply:
Aligned with the request above.